spec/system/welcome_spec.rb

```diff
@@ -59,6 +59,14 @@
         expect(page).not_to have_content "Please log in"
         expect(page).to have_content "Log Out"
       end
+
+      it "shows the Mediflux version on the home page for a logged in user" do
```

Comment:
This would change to:
```suggestion
      it "shows the Mediflux version on the home page for a logged in user", connect_to_mediaflux: true do
```